tpcx_bb/queries/q08/tpcx_bb_query_08_sql.py

```diff
@@ -236,6 +241,8 @@ def main(data_dir, client, bc, config):
     """
     result = bc.sql(last_query)
 
+    bc.drop_table("web_page_2")
```

Comment:
move ` bc.drop_table("web_page_2")` to right after `merged_df = bc.sql(query_2)` and add there a `del web_page_df`